docs/tables/hibp_account.md

```diff
@@ -1,26 +1,53 @@
 # Table: hibp_account
 
-This table returns data similar to the `hibp_breach` table, with the requirement of an `account` field.
+The most common use of the API is to return a list of all breaches a particular account has been involved in. This table returns data similar to the `hibp_breach` table, with the requirement and addition of an `account` field.
 
-While the `hibp_breaches` table will return all of the known breaches, there are millions of accounts associated with those
-breaches. To find the breaches associated with a certain account, you must use this table.
+While the `hibp_breaches` table will return all of the known breaches, this table can be used to find breaches for a particular account.
 
 ## Examples
```

Comment:
Can you please add 1-2 examples of showing how to join this table with another plugin's tables on email addresses? For instance, getting the email from the `ldap_user` table and then showing breached accounts, and another example getting users from `okta_user` as well.